### PR TITLE
Pluribus Networks cpu class module with unit tests

### PR DESCRIPTION
**Pull request by rajaspachipulusu17**
##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
pn_cpu_class.py module with unit tests.

##### ADDITIONAL INFORMATION
ansible 2.4.0.0
  config file = None
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.12 (default, Dec  4 2017, 14:50:18) [GCC 5.4.0 20160609]

```

### CODE REVIEW

**Comment by rajaspachipulusu17** [comment]
@Akasurde can you review the commits?

File: lib/ansible/modules/network/netvisor/pn_cpu_class.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    description:
+      - Target switch to run the CLI on.
+    required: False
+    type: str
+  state:
+    description:
+      - State the action to perform. Use 'present' to create cpu-class and
+        'absent' to delete cpu-class 'update' to modify the cpu-class.
+    required: True
+    choices: ['present', 'absent', 'update']
```
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/pn_cpu_class.py:34]: type is missing

File: lib/ansible/modules/network/netvisor/pn_cpu_class.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+  - This module can be used to create, modify and delete CPU class information.
+options:
+  pn_cliswitch:
+    description:
+      - Target switch to run the CLI on.
+    required: False
+    type: str
+  state:
+    description:
+      - State the action to perform. Use 'present' to create cpu-class and
```
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/pn_cpu_class.py]:
```suggestion
      - State the action to perform. Use C(present) to create cpu-class and
```

File: lib/ansible/modules/network/netvisor/pn_cpu_class.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+        'absent' to delete cpu-class 'update' to modify the cpu-class.
```
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/pn_cpu_class.py]:
```suggestion
        C(absent) to delete cpu-class C(update) to modify the cpu-class.
```

**Review by Akasurde — CHANGES_REQUESTED** [review]
Rest LGTM

**Comment by rajaspachipulusu17** [comment]
@Akasurde ansibot has some problem? It is in processing state from 40 mins.

**Review by pdam-arista — APPROVED** [review]

**Comment by rajaspachipulusu17** [comment]
@Akasurde  can you merge this if there is no issue? i have made all the requested changes.

Thanks,
Rajas

**Review by Akasurde — APPROVED** [review]

**Pull request merged**